packages/core/admin/admin/src/components/Notifications/index.js

```diff
@@ -1,11 +1,13 @@
-import React, { useEffect, useReducer } from 'react';
+import { NotificationsProvider } from '@strapi/helper-plugin';
+import React, { useReducer } from 'react';
+import PropTypes from 'prop-types';
 import { CSSTransition } from 'react-transition-group';
 
 import Notification from './Notification';
 import reducer, { initialState } from './reducer';
 import NotificationsWrapper from './Wrapper';
 
-const Notifications = () => {
+const Notifications = ({ children }) => {
```

Comment:
@HichamELBSI @mfrachet do you think we should move the component to the helper-plugin?

Reply:
In fact, I don't think it's needed anymore, putting this component in the helper plugin will only make the package heavier whereas, now with the hook and provider system de Notifications are available in the entire app by using a simple hook. I think it's better for now to leave it this way.